ignition/src/IgnitionRobot.cpp

```diff
@@ -333,6 +325,11 @@ bool IgnitionRobot::valid() const
     return true;
 }
 
+size_t IgnitionRobot::dofs() const
+{
+    return jointNames().size();
```

Comment:
Just to know, are we tracking somewhere that the joint contained in `jointNames` has 1 and only 1 DOFs?

Reply:
There are a lot of asserts in the code, not here specifically. I think we're well covered, but we should document (yet another thing to remember) that we only support 1 DoF joints.

Reply:
> (yet another thing to remember)

Yet another thing for which to open issues. : )

Reply:
I converted an old issue to collect all these links to TODO(cument) stuff https://github.com/robotology/gym-ignition/issues/37